scripts/!mods_preload/!rosetta.nut

```diff
@@ -410,6 +410,13 @@ def.addLang("ja", {
         return ::Const.Strings.EntityName[0] == "ネクロマンサー";
     }
 })
+def.addLang("zh_CN", {
```

Comment:
This meant to be 2 letter, this way will also work, but one will need to use

zh_CN = "..." 

in translation files. I guess I should decouple these two or provide a fixed "to" key.